index.html

```diff
@@ -15,6 +15,8 @@
         <h1 class="margin-10">Är det lördag idag?</h1>
       </div>
       <div class="result-row flex-row flex-center"></div>
+      <p class="flex-row flex-center">Idag är det <span id="day"></span></p>
+      <p id="message" class="flex-row flex-center"></p>
```

Comment:
I like how you have structured your code, however, you could benefit from learning about naming convention and when to use camelCase, and when to use kebab-case 👍